src/Lykke.Service.Assets.Core/Domain/IAssetPair.cs

```diff
@@ -19,5 +19,9 @@ public interface IAssetPair
         string Source { get; }
 
         string Source2 { get; }
+
+        double MinVolume { get; }
```

Comment:
Plz add a description for the new properties as they then used in the public API. src/Lykke.Service.Assets/Responses/v2/AssetPair.cs